src/issuer.ts

```diff
@@ -62,6 +63,7 @@ export class Issuer {
         {
           typ: Issuer.SD_JWT_TYP,
           alg: this.signer.alg,
+          ...header,
```

Comment:
Pls move header spread to top so that `typ` and `alg` can't be overridden ensuring adherence to the spec.

Reply:
shoudl be fine now